examples/plugins_advanced.md

```diff
@@ -0,0 +1,30 @@
+This is an example that installs a number of Elasticsearch plugins at run time, with specific settings for the given plugins using objects instead of strings.
+
+- Install latest available version of Elasticsearch
+- Name node *es-shield*
+- Install 3 plugins, including commercial Elastic plugin shield
+
+Run as `esvm plugins`
+
+Plugin objects can recieve one of the following:
+
+`path` - The file location of the given plugin, can be a file path or URL
+`staging` - Use the staged builds uploaded to S3
+`snapshot` - Use internal SNAPSHOT builds (requires credentials in `.esvm_snapshot_credentials.json`)
```

Comment:
Sory for the late response. The reference to the JSON file is in the main readme, which then links to the examples. 

Think we should be a little more clear about how the JSON file works in the readme?